### ISSUE TITLE
preTBC abusable items on level 70

### ISSUE THREAD

**Issue by Evolvee** [ccshiro/cc-buglist#2160]
Name(ID) :
1) Luffa(19141) - works fine on level 70
2) Tidal Charm(1404) - seems to have 100% chance against players level 70. (i couldnt test this much cuz of 15 mins cooldown) - 3 sec stun (should be around 50% chance)
3) Linkens Boomerang(11905) - have 100% chance against levels 70...    -||-   - 3 sec stun, 10 sec disarm - lol, I didnt find % on 2.4.3 tho, cant believe its 100%
 https://www.youtube.com/watch?v=vPqUxRVB4EE&feature=youtu.be
4) Hand of Justice - proc rate at level 70=1.33%,seems fine - it was 1.5% for me, but check this to be sure (it could be also set to 2%.. testing this is pure RNG)
5) Ultra-Flash Shadow Reflector + Hyper-Radiant Flame Reflector + Gyrofreeze Ice Reflector (18639 + 18638 + 18634)== ENG reflectors -- they have 100% chance to reflect, which is absurd

**Comment by DeeMarie** [comment]
http://wowwiki.wikia.com/wiki/Luffa

"Prior to The Burning Crusade, this item was not considered to be very valuable, however, its ability to dispel some bleed effects in the new end-game instances (e.g. Moroes's garrotte) made it very desirable. Unfortunately, patch 2.0.10 made the Luffa unable to dispel bleed effects above level 60, leaving little reason to save it after entering Outland

"

**Comment by Evolvee** [comment]
so? :D by "works fine on level 70" i meant it is not working at all ...

**Comment by DeeMarie** [comment]
Yes. I was just adding some evidence since I remember getting Luffa for Moroes then it not working, but other people might not remember it was fixed ;)

**Comment by ASrozaev** [comment]
tеstеd Tidal Charm
has great chancе to resist, so works finе
